src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponentSpec.scala

```diff
@@ -131,6 +131,18 @@ class SubmissionComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers
     }
   }
 
+  //if this unit test breaks, chances are you have added a submission to the test data which has changed the values below
```

Comment:
I don't like brittle tests.  We're going to be updating this one a lot as the test data changes, and that won't tell us much about whether these queries are working.

Could we use a different known data set?  Maybe a TestData which won't change, specifically for these.